webui/react/src/pages/ExperimentDetails/ExperimentDetailsHeader.tsx

```diff
@@ -296,36 +309,37 @@ const ExperimentDetailsHeader: React.FC<Props> = ({
               menu={[
                 {
                   key: 'Create New Experiment',
-                  label: 'Create New Experiment...',
+                  label: f_flat_runs
+                    ? 'Create New Run'
+                    : 'Create New Experiment...',
```

Comment:
nit: these should both or neither end in `...`, unless I'm missing that the Run version doesn't open a modal